.github/workflows/test.yaml

```diff
@@ -84,6 +84,26 @@ jobs:
       - name: Run integration tests
         run: mage test:integration
 
+  k8s-integration:
+    name: k8s Integration Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+
+      - name: Install tools
+        uses: aquaproj/aqua-installer@v2.1.1
+        with:
+          aqua_version: v1.25.0
```

Comment:
I don't think this test needs to install tools by aqua. Please let me know if it fails without this step.